pkg/watch/registrar.go

```diff
@@ -239,14 +240,14 @@ func (r *Registrar) ReplaceWatch(gvks []schema.GroupVersionKind) error {
 		}
 		roster[gvk] = wv
 	}
-	r.managedKinds.ReplaceRegistrarRoster(r, roster)
+	r.managedKinds.ReplaceRegistrarRoster(ctx, r, roster)
 	return r.mgr.replaceWatches(r)
 }
 
 // RemoveWatch removes a watch for the given kind.
 // Ignores the request if the kind was not previously watched.
-func (r *Registrar) RemoveWatch(gvk schema.GroupVersionKind) error {
```

Comment:
It looks like this context is no longer used?

Reply:
Done.